### ISSUE TITLE
FileHeader template differs from .config/irs.header

### ISSUE THREAD

**Issue by dsmf** [eclipse-tractusx/item-relationship-service#303]
https://github.com/catenax-ng/tx-item-relationship-service/blob/main/.idea/fileTemplates/includes/File%20Header.java
differs from
https://github.com/catenax-ng/tx-item-relationship-service/blob/main/.config/irs.header

![image](https://github.com/eclipse-tractusx/item-relationship-service/assets/19529362/7c3b181c-7060-4043-a8dc-554dd7226bb4)

## Hints
FileTemplate does not seem to be working in IntelliJ. Make sure the template is located in the correct folder and in the right structure.

**Comment by ds-jhartmann** [comment]
## Planning 2
- Make sure that the file header is up to date
- find out if the file template configuration is working in IntelliJ (fix if possible, remove if not)

**Comment by dsmf** [comment]
When I set up the project in IntelliJ during onboarding the "File Header.java" was not automatically configured in IntelliJ. It appeared under Settings / Editor / File and Code Templates but it was not applied when new files were created. I had to set it up via settings / Editor / Copyright and there it was not possible to just copy and paste the contents because here the comment signs have to be omitted. So we should investigate how to set this up, so it is used automatically.
